### ISSUE TITLE
Como le hare para subirlo a heroku?

### ISSUE THREAD

**Issue by Mataisplays** [ElVideoFanXd10/Slash-Guide#2]
Ya que para registrarlos es node slashcommands.js y luego node . , ¿como le hare para iniciarlos con 1 solo comando en la consola?

ya que si lo hago con **node slashcommands.js && node .**

sale esto: 
![image](https://user-images.githubusercontent.com/88283458/133907198-8a544fb0-793e-4a17-9ad3-680cf8217299.png)

ayuda :(

**Comment by ElVideoFanXd10** [comment]
> Ya que para registrarlos es node slashcommands.js y luego node . , ¿como le hare para iniciarlos con 1 solo comando en la consola?
> 
> ya que si lo hago con **node slashcommands.js && node .**
> 
> sale esto:
> ![image](https://user-images.githubusercontent.com/88283458/133907198-8a544fb0-793e-4a17-9ad3-680cf8217299.png)
> 
> ayuda :(

Hola!

Lamento informarte que  lo que intententas hacer no se puede, tienes que correrlos por separados

**Comment by Mataisplays** [comment]
Hola! Gracias por responder :), ab ria alguna forma? o tendre qeu buscar alguna otra alternativa?

**Comment by ElVideoFanXd10** [comment]
No, Lo tienes que hacer como lo dije primero node slashcommands.js y despues node .

**Comment by Mataisplays** [comment]
gracias por responder de nuevo :D por esa situación cambie un poco tu code para poder usarlo nada mas con node ., igualmente gracias por ayudarme y si quieres puedes cerrar el problema 😁
